build/static/boot/token.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+file_token="$CLUSTER_PATH/cluster_token.txt"
+
+if [[ -n "$DST_TOKEN" ]] && [[ ! -f "$file_token" ]]; then
```

Comment:
May we just call it `TOKEN` instead of `DST_TOKEN`? Do we have any other tokens?